package.json

```diff
@@ -36,6 +36,7 @@
     "lint": "eslint src/**/*.ts --fix",
     "lint:check": "eslint src/**/*.ts",
     "docs:install": "cd docs && yarn",
+    "docs:lint": "eslint -c docs/.eslintrc.cjs --ext .ts --ext .tsx docs/src ",
```

Comment:
question: are we calling this in our pipelines somewhere ?

Reply:
No, I don't think the pipeline specifically calls that command.  Simply, there were commands such as docs:install in package.json, so I thought it would be more convenient to have a lint command, so I wrote it there.  Is it okay to just list this only in the docs folder?